### ISSUE TITLE
is there a pypi version

### ISSUE THREAD

**Issue by sgratzl** [undefx/py3tester#2]
i.e. a public released version?

**Comment by undefx** [comment]
there's currently not a pypi package. would one be helpful? if so, i could look into publishing it.

**Comment by sgratzl** [comment]
it would be useful since then you can proper declare it as a dependency (e.g., in a requirement.txt) and don't need to clone it from the repo directory and then knowing that there is a source directory.

**Comment by undefx** [comment]
i agree it would be nice to have a pypi package for this. i don't have the
bandwidth to do this right now, but i'll leave this issue open tracking.
